### PR TITLE
[NOD-705] Add "total" field to paginated results

### PR DESCRIPTION
**Pull request by someone235**

### CODE REVIEW

File: kasparovd/controllers/block.go [hunk 1]
```diff
@@ -55,15 +55,22 @@ func GetBlockByHashHandler(blockHash string) (interface{}, error) {
 
 // GetBlocksHandler searches for all blocks
 func GetBlocksHandler(order string, skip uint64, limit uint64) (interface{}, error) {
-	if limit < 1 || limit > maxGetBlocksLimit {
```
Comment (stasatdaglabs) [inline kasparovd/controllers/block.go:58]: Please reconsider this implicit behavior and if you insist on keeping it, please document it.

File: kasparovd/controllers/block.go [hunk 2]
```diff
@@ -75,12 +82,20 @@ func GetBlocksHandler(order string, skip uint64, limit uint64) (interface{}, err
...
+	}
+
 	blockResponses := make([]*apimodels.BlockResponse, len(blocks))
 	for i, block := range blocks {
 		blockResponses[i] = convertBlockModelToBlockResponse(block)
 	}
-	return blockResponses, nil
+	return apimodels.PaginatedBlocksResponse{
+		Blocks: blockResponses,
+		Total:  count,
```
Comment (stasatdaglabs) [inline kasparovd/controllers/block.go]: Please choose either "total" or "count" and stick with it.

File: kasparovd/controllers/transaction.go [hunk 3]
```diff
@@ -101,9 +101,9 @@ func GetTransactionByHashHandler(txHash string) (interface{}, error) {
 // GetTransactionsByAddressHandler searches for all transactions
 // where the given address is either an input or an output.
 func GetTransactionsByAddressHandler(address string, skip uint64, limit uint64) (interface{}, error) {
-	if limit < 1 || limit > maxGetTransactionsLimit {
```
Comment (stasatdaglabs) [inline kasparovd/controllers/transaction.go:104]: Please reconsider this implicit behavior and if you insist on keeping it, please document it.

File: kasparovd/controllers/transaction.go [hunk 4]
```diff
@@ -137,7 +147,10 @@ func GetTransactionsByAddressHandler(address string, skip uint64, limit uint64)
 		txResponses[i] = convertTxDBModelToTxResponse(tx)
 		txResponses[i].Confirmations = rpcmodel.Uint64(confirmations(txResponses[i].AcceptingBlockBlueScore, selectedTipBlueScore))
 	}
-	return txResponses, nil
+	return apimodels.PaginatedTransactionsResponse{
+		Transactions: txResponses,
+		Total:        count,
```
Comment (stasatdaglabs) [inline kasparovd/controllers/transaction.go]: Please choose either "total" or "count" and stick with it.

**Review by stasatdaglabs — CHANGES_REQUESTED** [review]

**Review by stasatdaglabs — DISMISSED** [review]

**Review by stasatdaglabs — APPROVED** [review]

**Review by svarogg — APPROVED** [review]

**Pull request merged**